### PR TITLE
010 Validation in Livewire

### LINKED ISSUE #10: 010 Validation in Livewire
010 Validation in Livewire

### PR DESCRIPTION
**Pull request by aadhar41**
010 Validation in Livewire
@aadhargaur41 
@aadhar41 
#10

### CODE REVIEW

File: app/Livewire/CreatePoll.php [hunk 1]
```diff
@@ -42,8 +59,4 @@ public function createPoll()
 
         $this->reset(['title', 'options']);
     }
-
```
Comment (aadhargaur41) [inline app/Livewire/CreatePoll.php:45]: @aadhar41  is it not needed?

Comment (aadhar41) [reply app/Livewire/CreatePoll.php:45]: Extra commented code @aadhargaur41

Comment (aadhargaur41) [reply app/Livewire/CreatePoll.php:45]: Ok

**Review by aadhargaur41 — APPROVED** [review]
Approved @aadhar41

**Pull request merged**